ci/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
```

Comment:
do we also need to build the SimulatedPlayers?

Reply:
We do not - they don't have worker assemblies but are started in process by the coordinator (by adding a prefab)